CS342MemberViews/Sources/JenniferXu/JenniferXu.swift

```diff
@@ -0,0 +1,69 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+// Jennifer Xu Member View. Modeled partially after Paul Schmiedmayer's Member View.
+//
+// Cited:
+// GitHub Logo: GitHub Logos and Usage - https://github.com/logos
+// LinkedIn Logo: LinkedIn Brand Guidelines - https://brand.linkedin.com/downloads
+//
+//  Created by Jennifer Xu on 1/19/23.
+//
+
+import SwiftUI
+
+public struct JenniferXu: View {
+   @Binding var navigationPath: NavigationPath
+    
+   public var body: some View {
+       ScrollView(.vertical){
+           VStack(){
+               ProfileView()
+               
+               //Name and majors
+               VStack(spacing: 8) {
+                   Text("Jennifer Xu")
+                       .font(.title2.bold())
+                   Text("B.S Biomedical Computation\nM.S Computer Science")
+                       .font(.title3)
+               }
+               
+               //Social Media links
+               HStack(){
+                   SocialButton(social: "LI-In-Bug", url:"https://www.linkedin.com/in/jenniferxu01/", label: "linkedin")
+                   SocialButton(social: "github-mark", url:"https://github.com/jennxu23", label:"github" )
+                   SocialButton(url:"https://jennxu.netlify.app/", label: "personal")
```

Comment:
I'm not sure how to handle the licensing of these images. I pulled the icons from the official branding packages of LinkedIn and GitHub.